backend/app/models/review.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Review < ApplicationRecord
+  validates :rating, presence: true
```

Comment:
Probably worth a validation that rating is between 1 - 5? Otherwise you're in trouble if someone finds a way to rate a restaurant 10000....